PostFreeSurfer/scripts/CreateMyelinMaps.sh

```diff
@@ -110,20 +122,23 @@ fslmaths "$OutputMNIT1wImage" -abs "$OutputMNIT1wImage" -odt float
 fslmaths "$OutputMNIT1wImage" -div "$BiasFieldOutput" "$OutputMNIT1wImageRestore"
 fslmaths "$OutputMNIT1wImageRestore" -mas "$T1wMNIImageBrainMask" "$OutputMNIT1wImageRestoreBrain"
 
-applywarp --rel --interp=spline -i "$OrginalT2wImage" -r "$T1wImageBrainMask" -w "$OutputOrigT2wToStandard" -o "$OutputMNIT2wImage"
-fslmaths "$OutputMNIT2wImage" -abs "$OutputMNIT2wImage" -odt float
-fslmaths "$OutputMNIT2wImage" -div "$BiasFieldOutput" "$OutputMNIT2wImageRestore"
-fslmaths "$OutputMNIT2wImageRestore" -mas "$T1wMNIImageBrainMask" "$OutputMNIT2wImageRestoreBrain"
+if [ "${T2wPresent}" = "YES" ] ; then
```

Comment:
I'd still prefer for this `if` conditional to terminate after L130, and then start a new conditional after L132 for the "# Create T1w/T2w maps" section.  I know it is a bit redundant, but otherwise, you can get the impression that the "# Create T1w/T2w maps" section is somehow part of the MNI space section, which isn't the case.

Reply:
Done.